tasks/task01/Narcissistic_Number.py

```diff
@@ -0,0 +1,16 @@
+for k in range(1, 1000) :
+    if(k>=1 and k<10):
+        digit=k%10
+        if digit**1==k:
+         print ('Narcissistic__Number', k)
+    if(k>=10 and k<100):
```

Comment:
первый раз пишем if, потом, если условие продолжаеться - elif. То есть как-то так
```suggestion
    elif 10 <= k < 100:
```
http://pythonicway.com/python-conditionals